README.md

```diff
@@ -1,55 +1,23 @@
-<p><a href="https://circleci.com/gh/elastic/search-ui/tree/master"><img src="https://circleci.com/gh/elastic/search-ui/tree/master.svg?style=svg&circle-token=c637bc2af60035a1f4cb5367071999ced238be76" alt="CircleCI buidl"></a>
-  
-> Libraries for the fast development of modern, engaging search experiences. 🎉
+# [Elastic Search UI](https://www.elastic.co/enterprise-search/search-ui)
 
-# Elastic Search UI
+Search UI is a JavaScript search framework for implementing world-class search experiences without reinventing the wheel.
 
-A [React](https://reactjs.org) library that allows you to quickly implement search experiences without re-inventing the wheel.
+<img src="docs/images/screenshot.png">
 
-Check out our [full documentation site](https://docs.elastic.co/search-ui) which contains detailed documentation for setting up and using Search UI in your products.
+## Get started
 
-## Running Locally
+To quickly get started with Search UI, follow one of our tutorials:
+- [Search UI with Elastic App Search](https://docs.elastic.co/search-ui/tutorials/app-search)
+- [Search UI with Elasticsearch](https://docs.elastic.co/search-ui/tutorials/elasticsearch)
```

Comment:
I only added links to two tutorials here.
Workplace Search is not ready for production use and for site search use-cases we should probably point to the web crawler in App Search.

Reply:
Nice I like this.